demo-data/appointments.js

```diff
@@ -0,0 +1,249 @@
+"use strict";
```

Comment:
I think exports are not an es6 syntax.

Try using:

`export appointments = [...]`